data-sample/cmaq/form-ald2-aldx/form-ald2-aldx-alter-table.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+source ../../database.cfg
+
+# load cmaq-2010-2011.csv and cmaq_sample.sql to backend container:
+docker cp cmaq-sample-2011.csv ${DB_CONTAINER_NAME}:/cmaq-sample.csv
```

Comment:
Should cmap-sample-2010.csv file also need to be copied to container? That seems to be what the comment says, but only 2011 csv sample data is copied.

Reply:
For the development environment, no, it's scope is limited to a smaller pool of data which is currently only 2011.
For production we wouldn't be using docker for the database, and the scripting needs to be adjusted for the deployment to accommodate many different csv files of which the names are not known a priori.

Reply:
local dev environment only uses 2011 data at the moment.

Reply:
@mjstealey That was what I speculated as well. I think the comment is a bit misleading and it'd be great to update the comment to make it a bit clearer.